src/schemas/json/cargo.json

```diff
@@ -360,7 +360,7 @@
         }
       },
       "additionalProperties": false,
-      "x-tombi-table-keys-order": "version-sort"
+      "x-tombi-table-keys-order": "schema"
```

Comment:
In practice, this is a more complex sorting condition, where `rust` and `rustdoc` should appear first, followed by the peripheral tools sorted by version-sort.

A similar situation is defined in [features](https://github.com/SchemaStore/schemastore/blob/9984c525944733e91e6184d86e35ba870cb9468d/src/schemas/json/cargo.json#L2217). See [docs](https://tombi-toml.github.io/tombi/docs/json-schema#x-tombi-table-keys-order) for more details.